app/src/main/java/com/example/todokotlinandroid/CategoriesView.kt

```diff
@@ -0,0 +1,159 @@
+        CategoriesEmptyScreen()
+    } else {
+        CategoriesScreen(categoriesList = categoriesList, onCategoryClick = onCategoryClick)
+
+        }
+    }
+
+@OptIn(ExperimentalUnitApi::class)
+@Composable
+internal fun CategoriesScreen(
+    categoriesList: List<Category>,
+    onCategoryClick: () -> Unit
+) {
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.White)
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(
+                    start = dimensionResource(id = R.dimen.default_margin),
+                    top = 30.dp,
+                    end = 31.dp
+                )
+        ) {
+            Text(
+                text = "Categories",
+                fontWeight = FontWeight.Bold,
+                fontSize = TextUnit(32f, TextUnitType.Sp)
+            )
+            
+            Text(
+                text = "+",
+                fontWeight = FontWeight.Bold
+                )
+        }
+        LazyColumn(
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            items(categoriesList.size) { index ->
+                val category = categoriesList[index]
+                CategoryItemView(category)
+            }
+
+        }
+    }
+}
+
+@Composable
+private fun CategoryItemView(category: Category) {
+    Row(
+        horizontalArrangement = Arrangement.End,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(top = 8.dp)
+    ) {
+        Column(
+            modifier = Modifier
+                .background(LightRed)
+                .width(251.dp)
+                .clip(
+                    RoundedCornerShape(
+                        bottomStart = 10.dp,
+                        topStart = dimensionResource(id = R.dimen.corner_radius)
+                    )
+                )
+
+        ) {
+            Text(
+                modifier = Modifier
+                    .padding(
+                        start = dimensionResource(id = R.dimen.default_margin),
+                        top = dimensionResource(id = R.dimen.default_margin)),
+                text = "${category.name} (${category.totalItems})"
+            )
+
+            Row(
+                horizontalArrangement = Arrangement.End,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(
+                        top = 57.dp,
+                        end = dimensionResource(id = R.dimen.default_margin),
+                        bottom = dimensionResource(id = R.dimen.default_margin)
+                    )
+            ) {
+                Text(
+                    modifier = Modifier
+                        .padding(
+                            bottom = dimensionResource(id = R.dimen.default_margin),
```

Comment:
you have padding also on row above - that's why you have actually 32.dp instead of 16.dp. You can remove padding bottom from here or remove padding bottom from the parent Row

Reply:
same is for the end padding too